src/bin/add/args.rs

```diff
@@ -106,6 +109,22 @@ impl Args {
 
         Ok(vec![dependency])
     }
+
+    fn get_upgrade_prefix(&self) -> &'static str {
```

Comment:
I think to improve semantics, I would change `self.get_upgrade_prefix` to return an Option and map none as well as the "unknown flag" case to `None`.